### PR TITLE
The expiry of conversations can now be configured in the admin panel

### PR DESCRIPTION
**Pull request by annando**

### CODE REVIEW

File: mod/admin.php [hunk 1]
```diff
@@ -1562,6 +1564,7 @@ function admin_page_site(App $a)
...
 		'$dbclean_expire_days' 	=> ['dbclean_expire_days', L10n::t("Lifespan of remote items"), Config::get('system','dbclean-expire-days', 0), L10n::t("When the database cleanup is enabled, this defines the days after which remote items will be deleted. Own items, and marked or filed items are always kept. 0 disables this behaviour.")],
 		'$dbclean_unclaimed' 	=> ['dbclean_unclaimed', L10n::t("Lifespan of unclaimed items"), Config::get('system','dbclean-expire-unclaimed', 90), L10n::t("When the database cleanup is enabled, this defines the days after which unclaimed remote items (mostly content from the relay) will be deleted. Default value is 90 days. Defaults to the general lifespan value of remote items if set to 0.")],
+		'$dbclean_expire_conv' 	=> ['dbclean_expire_conv', L10n::t("Lifespan of raw conversation data"), Config::get('system','dbclean_expire_conversation', 90), L10n::t("The conversation data is used for ActivityPub and OStatus, as well as for debug purposes. It is save to remove it after some days, default is 90 days.")],
```
Comment (tobiasd) [inline mod/admin.php]: How many are "some days"? 2, 7. 14... Could you be more precise here or in the admin panel documentation?

Comment (annando) [reply mod/admin.php]: I'm currently unsure ;-)

File: mod/admin.php [hunk 2]
```diff
@@ -1562,6 +1564,7 @@ function admin_page_site(App $a)
...
 		'$dbclean_expire_days' 	=> ['dbclean_expire_days', L10n::t("Lifespan of remote items"), Config::get('system','dbclean-expire-days', 0), L10n::t("When the database cleanup is enabled, this defines the days after which remote items will be deleted. Own items, and marked or filed items are always kept. 0 disables this behaviour.")],
 		'$dbclean_unclaimed' 	=> ['dbclean_unclaimed', L10n::t("Lifespan of unclaimed items"), Config::get('system','dbclean-expire-unclaimed', 90), L10n::t("When the database cleanup is enabled, this defines the days after which unclaimed remote items (mostly content from the relay) will be deleted. Default value is 90 days. Defaults to the general lifespan value of remote items if set to 0.")],
+		'$dbclean_expire_conv' 	=> ['dbclean_expire_conv', L10n::t("Lifespan of raw conversation data"), Config::get('system','dbclean_expire_conversation', 90), L10n::t("The conversation data is used for ActivityPub and OStatus, as well as for debug purposes. It should be save to remove it after 14 days, default is 90 days.")],
```
Comment (MrPetovan) [inline mod/admin.php]: save => safe

I know it sounds the same in German, but guys.

Comment (annando) [reply mod/admin.php]: I should safe this discussion, to be save for the next time.

**Pull request merged**